src/ApiInterface/include/ApiInterface/Orbit.h

```diff
@@ -314,7 +314,7 @@ extern "C" {
 #endif
 
 // Material Design Colors #500
-typedef enum {
+typedef enum {  // NOLINT(modernize-use-using)
```

Comment:
Would it be possible to specify a reason? Something like
```
// NOLINT(modernize-use-using): This is C code.
```
should work, right?

Same below.